src/status_im/contexts/wallet/wallet_connect/session_proposal/view.cljs

```diff
@@ -157,7 +157,8 @@
    {:type                :no-title
     :background          :blur
     :icon-name           :i/close
-    :on-press            (rn/use-callback #(rf/dispatch [:navigate-back]))
+    :on-press            (rn/use-callback #(do (rf/dispatch [:navigate-back])
```

Comment:
(unrelated, but) it's weird we use :navigate-back here when it's a modal. Usually we use :dismiss-modal for session requests.

Reply:
This was made sometime ago. `naviagate-back` is not ideal. Dimiss modal is better, I'll fix it.

Reply:
still planning to change this, or someplace else?

Reply:
I changed this, maybe it got lost in rebase. I'll push again.